<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCzytelnik</class>
 <widget class="QDialog" name="DialogCzytelnik">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Czytelnik</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="3">
    <widget class="QLineEdit" name="lineEdit_nazwisko"/>
   </item>
   <item row="11" column="0">
    <widget class="QPushButton" name="pushButton_usun">
     <property name="text">
      <string>Usuń</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="lineEdit_imie"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Nazwisko</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLineEdit" name="lineEdit_pesel"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Wypożyczenia</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="4">
    <widget class="QTableView" name="wypozyczeniaTable">
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>150</number>
     </attribute>
    </widget>
   </item>
   <item row="11" column="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>PESEL</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Imię</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QPushButton" name="pushButton_zwroc">
     <property name="text">
      <string>Zwróć książkę</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="pushButton_wypozycz">
     <property name="text">
      <string>Nowe wypożyczenie</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogCzytelnik</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogCzytelnik</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_usun</sender>
   <signal>clicked()</signal>
   <receiver>DialogCzytelnik</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>437</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
